doc/README.md

```diff
@@ -0,0 +1,16 @@
+# Documentation
+
+This folder contains ready to read documentation. The 'src' folder contains the
+editable versions of these documents.
+
+## Creating graphs with graphviz
```

Comment:
I think this should go into the `doc/src` folder as it's documentation for the doc generation, not doc for the product

Reply:
Fixed with commit: b444abddfb738fa8e7adfbbb841b65953ebf6fcf